packages/react-component-library/src/components/Timeline/hooks/useTimelineZoom.ts

```diff
@@ -13,20 +12,10 @@ export function useTimelineZoom(): {
 } {
   const {
     dispatch,
-    state: { currentScaleIndex, options, scaleOptions, width },
+    state: { currentScaleIndex, scaleOptions },
   } = useContext(TimelineContext)
-  const [timelineScaleOptions, setTimelineScaleOptions] = useState(scaleOptions)
```

Comment:
This was trying to keep its own copy of the scale options. The ones in the context are kept up-to-date though, so rather than trying to fix this logic I just changed it to use the scale options directly from the context which seems to work fine as far as I can see.

Reply:
Ah nice, the code was changed recently and this addition makes it much cleaner 👍